### ISSUE TITLE
Translated/Stubbed BCL test variants

### ISSUE THREAD

**Issue by kg** [sq/JSIL#817]
Right now when the Stubbed BCL and Translated BCL variants of tests run, the compiled .dll and the output JS go into the same places (and overwrite previous outputs). This causes the individual test cases to share static field data when run natively, and it means that if you open up the output JS to debug a problem, you're not actually looking at the JS that was used.

For the DLL reuse thing that might be fine in the long run, or we can just solve that with appdomains.

For the JS output I'd like to tag the output JS somehow to indicate that it's from the stubbed bcl/translated bcl variant. The code should be the same most of the time (all of the time?) but I'd like to rule out that sort of issue.

Thoughts, @iskiselev ?

**Comment by iskiselev** [comment]
Good point. I've investigated it yet, but code could be different in this cases, as we have different overloaded method information in this cases (probably something more?).

**Comment by kg** [comment]
I've got one failure that isn't reliably reproducible as a result of this, I think:

```
// ListRemoveRange.cs ... 
failed: JavaScript interpreter exited with code 3 after throwing 1 exception(s):
JSIL.Tests.JavaScriptException: JavaScript Exception: 
TypeError: exc.GetType is not a function
Program_Main@C:\Users\Katelyn\Documents\Projects\JSIL\Tests\SimpleTestCases\ListRemoveRange.cs.out:85:25
runTestCase@C:\Users\Katelyn\Documents\Projects\JSIL\Libraries\JSIL.Shell.js:175:5
runMain@typein:5:201
shellStartup@C:\Users\Katelyn\Documents\Projects\JSIL\Libraries\JSIL.Shell.js:129:5
@typein:5:341
```

And a few failures that are clearly just from statics being reused (static ctor not running, value not being initialized how it was supposed to be)

**Comment by iskiselev** [comment]
Probably we should store translated test in subfolder (Namespace.TestClass.TestMetod). Will experiment with it tomorrow.
